astropy/wcs/wcsapi/tests/test_fitswcs.py

```diff
@@ -1139,13 +1213,13 @@ def test_spectral_1d(header_spectral_1d, ctype1, observer):
     else:
         header['CUNIT1'] = ''
 
-    header['RESTWAV'] = 1.420405752E+09
+    header['RESTWAV'] = 1.420405752e09
```

Comment:
This just feels like unnecessary. What is wrong with `1.420405752E+09`? Hard to read?

Reply:
Makes the code look like Fortran? 😉

Reply:
But it feels it is half-Fortran. To make it more Pythonic I would remove that unnecessary `0` stuck between `e` and `9`.

Reply:
I agree, think there is no reason we can't change to e9?